src/js/components/purchaseOrder/PurchaseOrderListTable.jsx

```diff
@@ -0,0 +1,385 @@
+      label: 'react.purchaseOrder.editLineItems.label',
+      defaultLabel: 'Edit line items',
+      leftIcon: <RiListUnordered />,
+      statuses: ['PENDING'],
+      activityCode: ['PLACE_ORDER'],
+      href: '/openboxes/purchaseOrder/addItems',
+    },
+    {
+      label: 'react.purchaseOrder.placeOrder.label',
+      defaultLabel: 'Place order',
+      leftIcon: <RiShoppingCartLine />,
+      statuses: ['PENDING'],
+      activityCode: ['PLACE_ORDER'],
+      href: '/openboxes/order/placeOrder',
+    },
+    {
+      label: 'react.purchaseOrder.printOrder.label',
+      defaultLabel: 'Print order',
+      leftIcon: <RiPrinterLine />,
+      activityCode: ['PLACE_ORDER'],
+      href: '/openboxes/order/print',
+    },
+    {
+      label: 'react.purchaseOrder.cancelOrder.label',
+      defaultLabel: 'Cancel order',
+      leftIcon: <RiCloseLine />,
+      activityCode: ['PLACE_ORDER'],
+      statuses: ['PLACED'],
+      href: '/openboxes/order/withdraw',
+    },
+    {
+      label: 'react.purchaseOrder.rollbackOrder.label',
+      defaultLabel: 'Rollback Order',
+      leftIcon: <RiArrowGoBackLine />,
+      minimumRequiredRole: 'Superuser',
+      activityCode: ['PLACE_ORDER'],
+      statuses: ['PLACED'],
+      href: '/openboxes/order/rollbackOrderStatus',
+    },
+    {
+      label: 'react.purchaseOrder.delete.label',
+      defaultLabel: 'Delete',
+      leftIcon: <RiDeleteBinLine />,
+      minimumRequiredRole: 'Assistant',
+      variant: 'danger',
+      onClickActionWithId: id => deleteOrder(id),
+    },
+  ];
+
+  // Columns for react-table
+  const columns = [
+    {
+      Header: 'Status',
+      accessor: 'status',
+      className: 'active-circle',
+      headerClassName: 'header',
+      style: { overflow: 'visible' },
+      Cell: row => (
+        <div className="d-flex gap-8">
+          <ActionDots actions={findActions(actions, row)} id={row.original.id} />
+          <PurchaseOrderStatus status={row.original.status} />
+        </div>
+      ),
+    },
+    {
+      Header: 'Order Number',
+      accessor: 'orderNumber',
+      className: 'cell',
+      headerClassName: 'header text-align-left',
+    },
+    {
+      Header: 'Name',
+      accessor: 'name',
+      className: 'cell',
+      headerClassName: 'header text-align-left',
+    },
+    {
+      Header: 'Supplier',
+      accessor: 'origin',
+      className: 'cell',
+      headerClassName: 'header text-align-left',
+    },
+    {
+      Header: 'Destination',
+      accessor: 'destination',
+      className: 'cell',
+      headerClassName: 'header text-align-left',
+    },
+    {
+      Header: 'Ordered On',
+      accessor: 'dateOrdered',
+      className: 'cell',
+      headerClassName: 'header text-align-left',
+      Cell: row => (
+        <span>{dateParser(row.original.dateOrdered)}</span>
+      ),
+      Footer: (
+        <span>
+          <Translate id="react.purchaseOrder.totalAmount.label" defaultMessage="Total amount" />: {totalPrice} USD
```

Comment:
Why it has hard coded `USD`? Add it in the DTO, taken from `grailsApplication.config.openboxes.locale.defaultCurrencyCode`